commands/commands.go

```diff
@@ -85,7 +85,7 @@ func FullError(err error) {
 
 func errorWith(err error, fatalErrFn func(error, string, ...interface{}), errFn func(string, ...interface{})) {
 	if Debugging || errors.IsFatalError(err) {
-		fatalErrFn(err, "")
+		fatalErrFn(err, "%s", err)
```

Comment:
Instead of having to do this, would it make more sense for `LoggedError` to just print the error contents if `format` is blank? Seems more intuitive.